src/main/java/com/prgrms/bdbks/domain/item/api/ItemController.java

```diff
@@ -29,7 +29,7 @@
 @RequiredArgsConstructor
 public class ItemController {
 
-	private final ItemFacadeService itemService;
+	private final ItemService itemService;
 
```

Comment:
컨트롤러에 간단한 명세를 적어보는건 어떨까요?
```suggestion
```suggestion
/**
	 * <pre>
	 *     충전 카드 단건 조회
	 * </pre>
	 * @param user - 조회할 충전카드의 User
	 * @param cardId - 조회할 충전카드의 Id
	 * @return status : ok, body : CardSearchResponse
	 */
```
```

Reply:
반영하도록 하겠습니다.

Reply:
수정했습니다!